rosidl_generator_py/resource/_msg.py.em

```diff
@@ -334,7 +334,18 @@ if isinstance(type_, AbstractNestedType):
         typename = self.__class__.__module__.split('.')
         typename.pop()
         typename.append(self.__class__.__name__)
+@[if 'import array' in imports]@
```

Comment:
Instead of using this runtime condition (which magically changes the logic for all fields as soon as an array field is added) use the types stored for each slot in `SLOT_TYPES`.

Reply:
I see, good point.  I've done this in 33e0ff68eac0e27030488b9b1e8683cc1ea063bc